src/crypto/x25519/mod.rs

```diff
@@ -549,19 +549,38 @@ fn mod_25519(x: Felem2) -> Felem {
 }
 
 fn mod_final_25519(x: Felem) -> Felem {
+    const MASK_64BITS: u128 = 0xffff_ffff_ffff_ffff;
```

Comment:
Defining this as a const at the top and replacing all instances would definitely be welcome, along with `0x7fff_...`.

Reply:
Good idea.